object_detection/g3doc/running_notebook.md

```diff
@@ -0,0 +1,14 @@
+# Quick Start: Jupyter notebook for off-the-shelf inference
+
+If you'd like to hit the ground running and run detection on a few example
+images right out of the box, we recommend trying out the Jupyter notebook demo.
+To run the Jupyter notebook, run the following command from
+`tensorflow/models/object_detection`:
+
+```
+# From tensorflow/models/object_detection
+jupyter notebook
+```
+
+The notebook should open in your favorite web browser. Click the
+`object_detection_tutorial.ipynb` link to open the demo.
```

Comment:
Add link to the notebook itself, github can show it.

Also add the instructions on how to run it to the notebook itself